easybuild/easyconfigs/j/Java/Java-1.8.0_152.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Java'
+version = '1.8.0_152'
+
+homepage = 'http://java.com/'
+
+description = """
```

Comment:
why the empty line?

Reply:
Hmm, just stolen from existing `Java` easyconfigs, will clean up.